src/main/java/com/eprosima/fastdds/idl/templates/TypeObjectTestingTestSource.stg

```diff
@@ -1260,6 +1264,7 @@ if (type_ids.type_identifier1()._d() == TI_PLAIN_MAP_SMALL)
         EXPECT_EQ(type_ids.type_identifier1().map_sdefn().header().equiv_kind(), EK_BOTH);
         EXPECT_TRUE(element_type_ids.type_identifier1() == *type_ids.type_identifier1().map_sdefn().element_identifier());
         EXPECT_TRUE(key_type_ids.type_identifier1() == *type_ids.type_identifier1().map_sdefn().key_identifier());
+        EXPECT_TRUE(element_type_ids.type_identifier2() == TK_NONE);
```

Comment:
In this and the case below, the key_element shall also have TK_NONE the second TypeIdentifier